pants.toml

```diff
@@ -28,3 +28,9 @@ plugins2 = []
 source_roots = """{
   '': ('python',),
 }"""
+
+
+[python-setup]
+# We have both Python 2 and Python 3 code in the repo, so we have to override the
+# default repo-wide value of CPython>=3.6.
+interpreter_constraints = "CPython>=2.7,!=3.0,!=3.1,!=3.2,!=3.3,!=3.4,!=3.5"
```

Comment:
I'd add something like this:

> This value means that all targets will default to working with both Python 2 and Python 3. Individual targets can set `compatibility` to indicate that they only work with Python 2 or only Python 3.

It's important to point out that a) you don't need to configure `compatibility` on every target and b) you likely want this default to be both Python 2 and Python 3. (Unless your code base is disproportionally Py2-only or Py3-only)

Reply:
Even if your codebase is disproportionately, say, Py3, don't you have to set this to py2+3, because the constraints are additive?

Reply:
In a mostly py3 repo is there a way to have the config default be py3, but have individual targets be py2? I thought they would get ANDed?

Reply:
That's the behavior I'm seeing on 1.27.0.dev0 I think

Reply:
Huh, I was wrong, not sure what I was seeing. Updating this appropriately.